modules/how-to-plan-your-environment-according-to-cluster-maximums.adoc

```diff
@@ -14,7 +14,7 @@ take to avoid memory swapping.
 
 [IMPORTANT]
 ====
-Some of the limits are stretched only in a single dimension. They will vary
+Some of the tested maximums are stretched only in a single dimension. They will vary
```

Comment:
ok looks like you are using the word "tested maximums". So at least in actual text that use reads, we should try to be consistent with that lang.